src/generators.js

```diff
@@ -175,9 +175,13 @@ PromiseSpawn.prototype._continue = function (result) {
             this._yieldedPromise = maybePromise;
             maybePromise._proxy(this, null);
         } else if (BIT_FIELD_CHECK(IS_FULFILLED)) {
-            this._promiseFulfilled(maybePromise._value());
+            Promise._async.invoke(
```

Comment:
It would be better to have `async` imported directly into the generators.js module